src/ledger/test/LedgerTxnTests.cpp

```diff
@@ -1355,6 +1355,16 @@ TEST_CASE("LedgerTxn load", "[ledgertxn]")
                         NonSociRelatedException);
                 }
 
+                SECTION("invalid asset")
+                {
+                    std::string assetCode;
+                    assetCode.push_back('\n');
+                    auto asset = txtest::makeAsset(acc, assetCode);
+                    REQUIRE_THROWS_AS(
+                        ltx1.load(trustlineKey(acc.getPublicKey(), asset)),
```

Comment:
This would have thrown anyway, because `tl.accountID = getIssuer(tl.asset)`. Can you fix this to make sure it is testing the relevant condition? A good way to guarantee this is to show that calling `getTrustLineStrings` directly doesn't throw for the given `asset` prior to protocol 15.

Reply:
Good point. Fixed.